### PR TITLE
Settings

### PR DESCRIPTION
**Pull request by chazr88**
# Description

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change Status

- [ ] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

- [ ] Test A
- [ ] Test B

# Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] My code has been reviewed by at least one peer
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] There are no merge conflicts

### CODE REVIEW

File: .vscode/settings.json
Comment (aalejandromr) [inline .vscode/settings.json:1]: Can you add .vscode/ to the .gitignore file, we shouldn't track this folder

File: src/components/Onboarding/index.js
Comment (aalejandromr) [inline src/components/Onboarding/index.js:5]: Would you please get rid of all unnecessary comments before merging?

File: src/store/actions/updateUserSettings.js
Comment (aalejandromr) [inline src/store/actions/updateUserSettings.js:6]: Code without need in this PR shouldn't be merge to staging. If it's not used it doesn't belong to this specific PR

**Review by JimmyMcBride — COMMENTED** [review]
Looks great! Let's get it merged!

File: .env
Comment (israelgonzalezb) [inline .env:1]: We need to remove .env files from the repo. This is a security risk because of the api keys and stuff that we throw in there.

File: src/components/Auth/Login/index.js
Comment (israelgonzalezb) [inline src/components/Auth/Login/index.js:55]: for the `isEmpty` var, is there a more descriptive variable name we could use here?

File: src/components/Auth/Register/RegisterOptions.js
Comment (israelgonzalezb) [inline src/components/Auth/Register/RegisterOptions.js:106]: Is this something we need in our global styled components folder?

File: src/components/DailyLog/index.js
Comment (israelgonzalezb) [inline src/components/DailyLog/index.js:56]: Do these belong in our global styled folder?

File: src/components/Global/styled/components/Form.js
Comment (israelgonzalezb) [inline src/components/Global/styled/components/Form.js:3]: TODO?

File: src/components/Onboarding/components/DietaryGoal.js
Comment (israelgonzalezb) [inline src/components/Onboarding/components/DietaryGoal.js:15]: Where is this being used in this file?

File: src/components/Settings/components/modal.js
Comment (israelgonzalezb) [inline src/components/Settings/components/modal.js:36]: What's supposed to go here?

File: src/components/Settings/index.js
Comment (israelgonzalezb) [inline src/components/Settings/index.js:78]: Will these be used anywhere else? If so, we can put them in the global styled folder 😁

File: src/components/Settings/styles.js
Comment (israelgonzalezb) [inline src/components/Settings/styles.js:10]: This code is duplicated in components/Settings/index.js

File: src/store/actions/updateUserSettings.js
Comment (israelgonzalezb) [inline src/store/actions/updateUserSettings.js:6]: TODO

**Review by israelgonzalezb — APPROVED** [review]
All good, my comments are suggestions. I know some things are still WIP while we integrate backend with frontend. Awesome job on the settings!!

**Comment by dave-vazquez** [comment]
❗️ Hold off merging this. There are other branches pulled into this PR that haven't been reviewed yet.

**Pull request closed without merging**
